runelite-client/src/main/java/net/runelite/client/plugins/timersandbuffs/TimersAndBuffsPlugin.java

```diff
@@ -488,16 +487,11 @@ else if (nextSuperAntifireTick - tickCount <= 0)
 				// However, after the first tick, Varbits.MOONLIGHT_POTION is suddenly transmitted BEFORE
 				// Varbits.DIVINE_SUPER_DEFENCE every tick. Thus, when the latter is transmitted, the
 				// Varbits.MOONLIGHT_POTION value might be 1 less than the Varbits.DIVINE_SUPER_DEFENCE value.
-				|| (showMoonlightPotion && client.getVarbitValue(Varbits.MOONLIGHT_POTION) + 1 >= event.getValue()))
+				|| (config.showMoonlightPotion() && client.getVarbitValue(Varbits.MOONLIGHT_POTION) + 1 >= event.getValue()))
```

Comment:
The order of these checks should be flipped; `getVarbitValue()` is more or less free, calls to config are not.